jwtmiddleware/middleware.go

```diff
@@ -89,35 +101,57 @@ func NewMiddleware(
 	return &Middleware{
 		o:   c,
 		wku: wellKnownURL,
-		jm: jwtmiddleware.New(jwtmiddleware.Options{
-			ValidationKeyGetter: func(token *jwt.Token) (interface{}, error) {
-				if raw, ok := token.Header["kid"]; !ok {
-					return nil, errors.New(`jwt from authorization HTTP header is missing value for "kid" in token header`)
-				} else if kid, ok := raw.(string); !ok {
-					return nil, fmt.Errorf(`jwt from authorization HTTP header is expecting string value for "kid" in tokenWithoutKid header but got: %T`, raw)
-				} else if k, err := jc.GetKey(kid); err != nil {
-					return nil, err
-				} else {
-					return k.Key, nil
-				}
+		jm: jwtmiddleware.New(
+			func(ctx context.Context, rawToken string) (any, error) {
```

Comment:
Is debug missing now?

Reply:
Yes, the debug option does not exist anymore. Instead, the errors returned should have more context.